src/components/sbb-timetable-occupancy-icon/sbb-timetable-occupancy-icon.scss

```diff
@@ -0,0 +1,23 @@
+@use '../core/styles' as sbb;
+
+// Default component properties, defined for :host. Properties which can not
+// travel the shadow boundary are defined through this mixin
+@include sbb.host-component-properties;
+
+:host {
+  --sbb-icon-svg-height: #{sbb.px-to-rem-build(12)};
```

Comment:
I saw that you overwrote this from the train component. So maybe it requires a property as also consumers need to have this maybe?

Reply:
Done